mesh/mesh.hpp

```diff
@@ -1150,21 +1223,29 @@ class Mesh
    Geometry::Type GetBdrElementBaseGeometry(int i) const
    { return GetBdrElementGeometry(i); }
 
-   /** @brief Return true iff the given @a geom is encountered in the mesh.
-       Geometries of dimensions lower than Dimension() are counted as well. */
-   bool HasGeometry(Geometry::Type geom) const
-   { return mesh_geoms & (1 << geom); }
+   /// Return true if the given face is interior. @sa FaceIsTrueInterior().
+   bool FaceIsInterior(int FaceNo) const
+   {
+      return (faces_info[FaceNo].Elem2No >= 0);
+   }
 
-   /** @brief Return the number of geometries of the given dimension present in
-       the mesh. */
-   /** For a parallel mesh only the local geometries are counted. */
-   int GetNumGeometries(int dim) const;
+   /** @brief Get the size of the i-th element relative to the perfect
```

Comment:
What does type do here?  Do we have the specify the element type in this method?